### PR TITLE
[toolbar] Merged colored icons in toolbar

### PR DESCRIPTION
**Pull request by jbgomond**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets |  |
| License | MIT |
| Doc PR |  |

Status icons are doubled without any apparent reason.

Token before
capture_token_before

Token after
capture_token_after

Security before
capture_security_before

Security after
capture_security_after

### CODE REVIEW

**Comment by phansys** [comment]
:+1:
This is good for saving toolbar space.
BTW, you should squash your commits.

**Comment by peterrehm** [comment]
@phansys The commits will be squashed latest once the PR is being merged, this is part of the `gh` toolchain Fabien showed once.

**Comment by phansys** [comment]
I've thought in the whole change as a single one, but you're right if you are trying to  track each change individually. Thank you for the clarification, I didn't know about the toolchain.

**Comment by peterrehm** [comment]
@phansys Once it is being merged it will be squashed into one commit if you have not squashed it before in order to maintain a clean history

File: src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig [hunk 1]
```diff
@@ -32,8 +32,7 @@
...
-        <span class="sf-toolbar-status sf-toolbar-status-{{ color_code }}"></span>
-        {% if collector.user %}<div class="sf-toolbar-status sf-toolbar-info-piece-additional">{{ collector.user }}</div>{% endif %}
+        <span class="sf-toolbar-status sf-toolbar-status-{{ color_code }}">{{ collector.user|default('') }}</span>
```
Comment (stof) [inline src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig]: no need for ``|default('')`` here. The ``getUser`` method always exists on the collector, and displaying an empty string as default when the user is empty is useless. Just display the empty user name.

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/config.html.twig [hunk 2]
```diff
@@ -69,8 +69,7 @@
...
     {% set icon %}
         <svg width="21" height="28" xmlns="http://www.w3.org/2000/svg" version="1.1" x="0px" y="0px" viewBox="0 0 21 28" enable-background="new 0 0 21 28" xml:space="preserve"><g><path fill="#3F3F3F" d="M5 17H1c-0.5 0-1-0.4-1-1V9c0-0.5 0.4-1 1-1h4c0.5 0 1 0.4 1 1v7C6 16.6 5.6 17 5 17z"/><path fill="#3F3F3F" d="M19 17h-4c-0.5 0-1-0.4-1-1V9c0-0.5 0.4-1 1-1h4c0.5 0 1 0.4 1 1v7C20 16.6 19.6 17 19 17z"/><path fill="#3F3F3F" d="M12.1 20h-4c-0.5 0-1-0.4-1-1v-7c0-0.5 0.4-1 1-1h4c0.5 0 1 0.4 1 1v7C13.1 19.5 12.6 20 12.1 20z"/><rect x="9.1" y="5.8" fill="#3F3F3F" width="2" height="4.5"/><rect x="16" y="17.7" fill="#3F3F3F" width="2" height="4.5"/><rect x="2.1" y="17.7" fill="#3F3F3F" width="2" height="4.5"/><rect x="9.1" y="20.7" fill="#3F3F3F" width="2" height="1.5"/><rect x="2.1" y="5.7" fill="#3F3F3F" width="2" height="1.5"/><rect x="16.1" y="5.7" fill="#3F3F3F" width="2" height="1.5"/></g></svg>
-        <span class="sf-toolbar-info-piece-additional-detail {{ debug_status_class }}"> </span>
-        <span class="sf-toolbar-info-piece-additional sf-toolbar-status">{{ token }}</span>
+        <span class="{{ debug_status_class }}">{{ token }}</span>
```
Comment (stof) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/config.html.twig]: you should keep the ``sf-toolbar-info-piece-additional`` class IMO

**Comment by MJBGO** [comment]
Updated

**Comment by stof** [comment]
@fabpot please fix fabbot.io to avoid having such error messages

**Comment by stof** [comment]
:+1:

File: src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig [hunk 3]
```diff
@@ -32,8 +32,7 @@
...
-        <span class="sf-toolbar-status sf-toolbar-status-{{ color_code }}"></span>
-        {% if collector.user %}<div class="sf-toolbar-status sf-toolbar-info-piece-additional">{{ collector.user }}</div>{% endif %}
+        <span class="sf-toolbar-status sf-toolbar-status-{{ color_code }}">{{ collector.user }}</span>
```
Comment (aitboudad) [inline src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig]: you should keep the if statement for `collector.user` 

```twig
<span ...>{% if collector.user %}{{ collector.user }}{% endif %}</span>
```

**Comment by aitboudad** [comment]
:+1:

File: src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig (hunk 3 above)
Comment (jakzal) [inline src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig]: No need. It will return an empty string if there's no user.

Comment (aitboudad) [inline src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig]: right :)

**Comment by xabbuh** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @MJBGO.

**Pull request closed without merging**